src/main/java/clients/SymBotClient.java

```diff
@@ -168,10 +168,17 @@ private SymBotClient(SymConfig config, ISymAuth symBotAuth) {
         this.podClient = HttpClientBuilderHelper.getHttpClientBuilderWithTruststore(config).withConfig(podConfig).build();
         this.agentClient = HttpClientBuilderHelper.getHttpClientBuilderWithTruststore(config).withConfig(agentConfig).build();
 
-        this.botUserInfo = parseUserFromSessionToken(symBotAuth.getSessionToken());
-
+        try {
+            this.botUserInfo = parseUserFromSessionToken(symBotAuth.getSessionToken());
+        }
+        // Exception happens when you can not get userId or username out of the JWT
+        // When that happens grab account info via API.
+        catch (ArrayIndexOutOfBoundsException e) {
+            getBotUserInfo();
```

Comment:
Add a debug log before this line, saying that something like "Using API to retrieve user info"